### DISCUSSION TITLE
Why do you specify a newsgroup in your Category settings?

### DISCUSSION THREAD

**Discussion by Safihre** [sabnzbd/sabnzbd#2758] [General]
Looking for use-cases, otherwse this will be removed.
Related also to #2715,

# Seems we are keeping this functionality! 🥳

**Comment by HaArD2000** [comment]
It helps to assign the right category to NZBs automatically when the indexer doesn't supply a meta tag in the nzb file. Many newsgroups are still organized by content type and this is a reliable means of identifying the type of content being downloaded.

**Reply by Safihre** [reply]
I didn't expect indexers out there to still not supply a category. 
But if that's the case, then we of course keep this functionality to match by group.

**Comment by thezoggy** [comment]
To confirm, I believe saf is asking about those that use the usenet binaries group (example: `alt.binaries.teevee`) to match off instead of just using the usenet indexer tag or such.

for example a mp3 .nzb I grabbed from a site is posted across like 10+ groups,  
```
    <groups>
      <group>alt.binaries.comp</group>
    </groups>
    ...
    <groups>
      <group>alt.binaries.ftn</group>
    </groups>
    ...
    <groups>
      <group>alt.binaries.vcd</group>
    </groups>
    ...
    <groups>
      <group>alt.binaries.boneless</group>
    </groups>
```

we extract all those groups and make a list, then use the first one that the users category group matches

**Reply by HaArD2000** [reply]
I understood the question and my response stands.  :-)

**Comment by DarrenPIngram** [comment]
Unrelated note concerning categories [since I was tracking down an issue that I thought was "user error" in the week]. Might it be worthy adding a note in the documentation for indexers that do not support categories directly? For example [redacted as thezoggy is aware of the target] does not [confirmed by support at least this week in messaging]. Their NZB shows the NG, but no internalised category matching such as TV.

**Reply by thezoggy** [reply]
let me see if i can get that addressed, as it should have mimics how nn does it which includes that meta tag.. 
confirmed with site, will be added. no eta

**Comment by ciaw** [comment]
Yes, please keep this function. I have a service that posts their stuff in a specific usenet group, that is not indexed/categorised by newznab sites, and so by tagging it at the group level I can have SAB auto sort their stuff correctly onto a specific drive.

**Comment by klatch29** [comment]
I specify a newsgroup in my categories as when I first started using sab for some manual downloads I don't think certain things I was sending were getting categorized properly from the indexer, or I was perhaps doing searches outside of an indexer. Anyways, it seemed to help ensure things got categorized properly so they could be handled properly and go to the right folder path afterwards. Honestly not sure if it would still be needed for my personal use case.

**Comment by elecgnosis** [comment]
Neither binsearch nor nzbindex use categories, so when I use their NZBs, I need some way to organize the content. Using the group name is how I get it done.

**Comment by b0xhub** [comment]
use this for reasons described above as well.

**Comment by mchangsp** [comment]
I use categories to store files downloaded from a specific newsgroup in a specific folder on disk.

**Comment by Safihre** [comment]
Thank you all. We are keeping this functionality!
